crates/aptos/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 All notable changes to the Aptos CLI will be captured in this file. This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) and the format set out by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+- **Breaking Change** Update View functions to use BCS for submission.  Allows for all arguments to be supported in view functions.  Note some input arguments that were previously inputted as strings may be handled differently.
```

Comment:
Might be worth calling out what the user needs to do differently.

Reply:
Might be missing something, but why is this a breaking change? It seems like we're not expecting any negative impact, we're just making it more accurate?

Reply:
I don't expect any negative impact, but it may have incompatibilities / unexpected behaviors that come with before passing in raw bytes or a string, that aren't how it worked before.

All arguments before were in string form, and I'm not sure how most of them worked, just covering the bases here.